package.json

```diff
@@ -1,373 +1,388 @@
 {
```

Comment:
Sorry about the changes in this file. I ran npm install --save for a particular package and it reformatted the file.